principles/fp-011-locus-of-authority.md

```diff
@@ -15,23 +15,34 @@ light of scientific advance, and for ensuring that all user feedback is addresse
 
-## Implementation
-    
-### Recommendations and Requirements
-
-The name, email address and github username of this person should be provided when requesting to register with [OBO](http://obofoundry.org). 
-This person must be subscribed to 
-obo-discuss in order to be kept abreast of community developments of relevance to 
-participating ontology projects. The email address of the person who is the locus of the 
-authority must be kept up-to-date, and before that person ceases to have responsibility 
+## Recomendations and Requirements
+A primary contact person MUST be assigned.
+The name, email address and GitHub username of the contact person MUST be provided when requesting to register with [OBO](http://obofoundry.org). The contact person MUST be subscribed to obo-discuss in order to be kept abreast of community developments of relevance to 
+participating ontology projects, but the primary contact person can, of course, delegate 
+these responsibilities for the project as necessary. The email address of the person who is the locus of the 
+authority MUST be kept up-to-date, and before that person ceases to have responsibility 
 for the project, they should identify a replacement and update the metadata accordingly 
-(via the [OBO Foundry issue tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues)) before they move on. The contact person can, of course, delegate 
-these responsibilities as necessary.
+(via the [OBO Foundry issue tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues)) before they move on.
+
+## Implementation
+
+First, read the [FAQ](http://obofoundry.github.io/faq/how-do-i-edit-metadata.html) on how to edit the metadata for your ontology.
+
+The contact email MUST NOT be a mailing list. If this person does not already have a GitHub account, we request that they [create one](https://github.com/join). Then, add the following to your [metadata file](https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology) (replacing with the correct email, name, and GitHub username):
+
+`contact:
+ email: foo@bar.com
+ label: John Smith
+ github: jsmith123`
 
 ### Examples: 
 For ChEBI, the primary contact person is Adnan Malik (amalik@ebi.ac.uk). 
```

Comment:
and his github tag?

Reply:
will do! I will change this example to Mondo and add my contact info.